phing/tasks/acsf.xml

```diff
@@ -7,12 +7,33 @@
     <echo>In future, you may use pass a custom value for acsf.package to override default version. E.g., blt acsf:init -Dacsf.package='drupal/acsf:8.1.x-dev'</echo>
     <exec dir="${repo.root}" command="composer require ${acsf.package}" logoutput="true" checkreturn="true" passthru="true" level="${blt.exec_level}"/>
     <!-- Run the acsf init commands. -->
-    <phingcall target="acsf:init:drush"/>
+    <phingcall target="acsf:init:all" />
+  </target>
+
+  <!-- Perform acsf:init:site for each multisite. -->
+  <target name="acsf:init:all" description="Refreshes the acsf settings and hook files for all acsf sites.">
+    <foreach list="${multisite.names}" param="site.name" target="acsf:init:site"/>
+  </target>
+
+  <!-- Wrapper around acsf:init:site:drush. -->
+  <target name="acsf:init:site" description="Refreshes the acsf settings and hook files for an acsf site.">
```

Comment:
I'm not certain that this is necessary as I'm not entirely familiar with the requirements of ACSF. This can be reverted if initialization doesn't need to be done on a per-site basis.